src/main/java/net/dv8tion/jda/api/events/thread/member/ThreadMemberLeaveEvent.java

```diff
@@ -17,16 +17,48 @@
 package net.dv8tion.jda.api.events.thread.member;
 
 import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.entities.Member;
 import net.dv8tion.jda.api.entities.ThreadMember;
 import net.dv8tion.jda.api.entities.channel.concrete.ThreadChannel;
 
 import javax.annotation.Nonnull;
 
-//TODO-v5: Docs
+/**
+ * Indicates that a guild {@link Member} left a {@link ThreadChannel}.
+ *
+ * @see ThreadChannel
+ * @see ThreadMember
+ */
 public class ThreadMemberLeaveEvent extends GenericThreadMemberEvent
 {
     public ThreadMemberLeaveEvent(@Nonnull JDA api, long responseNumber, ThreadChannel thread, long threadMemberId, ThreadMember threadMember)
     {
         super(api, responseNumber, thread, threadMemberId, threadMember);
     }
+
+    /**
+     * The left {@link ThreadMember}
+     * @return The left {@link ThreadMember}
+     */
```

Comment:
These docs (And the join event docs) need improvement.
```suggestion
    /**
     * The {@link ThreadMember} that just left the thread.
     * This entity will not be present in {@link ThreadChannel#getThreadMembers()} list.
     * 
     * @return The {@link ThreadMember} that just left the thread
     */
```

Reply:
Not sure if it was unclear, but this applies to bunch of the methods in this file and also to the Join event file